packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -224,3 +233,77 @@ function Example() {
     </Flex>
   );
 }
+
+function Pending(props) {
+
+  return (
+    <>
+      <View backgroundColor={props.variant === 'overBackground' ? 'static-blue-700' : undefined} padding={16}>
+        <PendingButtonComponent {...props}>click me!</PendingButtonComponent>
+      </View>
+
+      <View backgroundColor={props.variant === 'overBackground' ? 'static-blue-700' : undefined} padding={16}>
+        <PendingButtonComponent
+          {...props}>
+          <Bell />
+          <Text>I have an icon</Text>
+        </PendingButtonComponent>
+      </View>
+
+      <View backgroundColor={props.variant === 'overBackground' ? 'static-blue-700' : undefined} padding={16}>
+        <PendingButtonOnClickComponent
+          {...props}>
+          <Text>with onClick</Text>
+        </PendingButtonOnClickComponent>
+      </View>
+
+      <View padding={16}>
+        <Button variant="primary" isPending={props.isPending} onPress={() => {window.alert('use storybook control to change this button isPending prop');}}>
+          <Text>Controlled pending</Text>
+        </Button>
+      </View>
```

Comment:
don't need this if we just hook up all of the buttons to the controls

Reply:
All the examples are hooked up to controls for isPending except three of the ones in the Pending story. I wanted to have clickable examples but I didn't want anyone to miss the externally controlled version either.